lib/internal/Magento/Framework/HTTP/ClientInterface.php

```diff
@@ -105,6 +113,24 @@ public function get($uri);
      */
     public function post($uri, $params);
 
+    /**
+     * Make PATCH request
+     *
+     * @param string $uri full uri
+     * @param array|string $params PATCH fields array or string in case of JSON or XML data
+     * @return void
+     */
+    public function patch($uri, $params);
```

Comment:
This is MAJOR breking changes to SPI
Better to move it to separate interfases

btw, include PSR/HttpClient will be best option

Reply:
Not sure what you want me to do here.

Just remove changes from Interface ? Separate interface = ?

Major releases exist for pushing major BC, can't we just wait for it ? (And those who need this quickly can just use a patch, as I did)